api/block/routes.go

```diff
@@ -130,6 +172,15 @@ func getQueryParamNonce(c *gin.Context) (uint64, error) {
 	return strconv.ParseUint(nonceStr, 10, 64)
 }
 
+func getQueryParamRound(c *gin.Context) (uint64, error) {
+	roundStr := c.Param("round")
+	if roundStr == "" {
```

Comment:
lines L177 - L179 can be safely removed

Reply:
Removed it from here as well as from `getQueryParamNonce`